packages/components/src/Inject/Inject.test.js

```diff
@@ -165,6 +165,15 @@ describe('Inject.all', () => {
 		};
 		expect(Inject.all(null, components)()).toEqual(null);
 	});
+	it('should return a function which return a react component', () => {
+		const getComponent = jest.fn();
```

Comment:
please add expect it has not been called

Reply:
I don't think this is necessary. You just created it here, without passing it to something that is executed.
Checking that it has not been called here would be equivalent to 
```javascript
const obj = { value: 'lol' };
expect(obj.value).toBe('lol'); // not really useful

execute(obj);
expect(obj.value).toBe('other');
```